trino-connector/trino-connector/src/main/java/org/apache/gravitino/trino/connector/catalog/iceberg/IcebergDataTypeTransformer.java

```diff
@@ -20,6 +20,9 @@
 package org.apache.gravitino.trino.connector.catalog.iceberg;
 
 import io.trino.spi.TrinoException;
+import io.trino.spi.type.TimeType;
+import io.trino.spi.type.TimestampType;
+import io.trino.spi.type.TimestampWithTimeZoneType;
 import io.trino.spi.type.VarbinaryType;
 import io.trino.spi.type.VarcharType;
 import org.apache.gravitino.rel.types.Type;
```

Comment:
seems you shoud add corresponding logic to convert MICROS time to Iceberg time, and forbidden other time precision.

Reply:
please refer to https://github.com/trinodb/trino/blob/457/plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/TypeConverter.java#L148-L177

Reply:
Trino  iceberg  connector can automatically convert other time precision to MICROS

Reply:
what will happen if the user creates an Iceberg table with `timestamp(3)` ?

Reply:
It automatically converts to `timestamp(6)`, The behavior is the same as in the Trino Iceberg catalog.

Reply:
according to the code https://github.com/trinodb/trino/blob/457/plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/TypeConverter.java#L148-L177,  it will throw an exception in origin Trino Iceberg catalog, Yes?

Reply:
Yes

Reply:
so I don't think it's the right behavior to convert  `timestamp(3)` to `timestamp(6)` in Gravitino Iceberg connector, it conflict with original behavior.

Reply:
Trino Iceberg connector can  automatically converts  timestamp and timestamp(n) to timestamp(6), 
The Gravitino connector Iceberg catalog's  behavior is the same as Trino Iceberg connector